draft-ietf-taps-interface.md

```diff
@@ -2301,8 +2305,9 @@ Default:
 
 This property specifies that a message should be sent and received as a single
 packet without network-layer fragmentation, if possible.
-Attempts to send a message with this property set with a size greater to the
-transport's current estimate of its maximum transmission segment size will
+Attempts to send a message with this property set to a size greater than the
+transport's current estimate of its maximum packet segment size (`singularTransmissionMsgMaxLen`)
```

Comment:
"maximum packet segment size" sounds like a strange concept. Is it a packet or a segment? Actually segment most commonly refers to TCP, doesn't it... being general, maybe this word shouldn't be used?